sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_session.py

```diff
@@ -30,7 +30,7 @@
 class BaseSession(object):
     def __init__(self, session_id, receiver, encoding="UTF-8"):
         # type: (str, Union[ServiceBusSessionReceiver, ServiceBusSessionReceiverAsync], str) -> None
-        self._session_id = session_id
+        self._id = session_id
```

Comment:
why rename this? session_id seems less vague

Reply:
Fun story: session_id was going to be id, but for some last minute arch discussion.  I didn't think this through when changing it back, and left the internal as _id.  Good callout.  If I get other feedback that would trigger a retest I'll do this now, otherwise I'll do this and the other minor change as an issue for oct so we can get this in in time.

Reply:
as above https://github.com/Azure/azure-sdk-for-python/issues/13686 (slipping this PR in under the wire and moving nonblocking changes to oct)